doc/classes/AudioEffectGate.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioEffectGate" inherits="AudioEffect" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Adds a gate to the audio bus to silence quiet audio.
+	</brief_description>
+	<description>
+		Silences audio when it is below a certain volume. Useful for processing microphones with background noise.
+	</description>
+	<tutorials>
+		<link title="Audio buses">$DOCS_URL/tutorials/audio/audio_buses.html</link>
+	</tutorials>
+	<members>
+		<member name="attack_ms" type="float" setter="set_attack_ms" getter="get_threshold_ms" default="5.0">
+			Amount of time it takes the gate to go from 0% to 100% volume when audio goes above the threshold.
+		</member>
+		<member name="hold_ms" type="float" setter="set_hold_ms" getter="get_hold_ms" default="100.0">
+			Amount of time it takes after audio goes below the threshold for the release envelope to start.
+		</member>
+		<member name="release_ms" type="float" setter="set_release_ms" getter="get_release_ms" default="50.0">
+			Amount of time it takes the gate to go from 100% to 0% volume after the gate is done holding.
+		</member>
+		<member name="threshold_db" type="float" setter="set_threshold_db" getter="get_threshold_db" default="-50.0">
+			Threshold below which the audio is silenced, in decibels. Value can range from -100 to 0.
```

Comment:
-100 to 0 portion is incorrect? It currently say the default is -50 which implies confusion between decibels and percent. Godot uses decibels.

Reply:
wha

Reply:
I misread as 100.